scripts/get-orig-tarball

```diff
@@ -63,10 +71,32 @@ is_released() {
     [ "$1" != "UNRELEASED" -a "$1" != "unreleased" ]
 }
 
-get_dsc_url() {
-    local name="$1" ver="$2"
+check_url() {
+    wget --method=HEAD --quiet "$url"
+}
+
+# get_dsc_urls(package, version)
+get_dsc_urls() {
+    local name="$1" ver="$2" url="" letter1="" comp="" dbase=""
     ver=$(no_epoch "$ver")
-    _RET="${DSC_BASE_URL}/${name}_${ver}.dsc"
+    url="${DSC_BASE_URL}/${name}_${ver}.dsc"
+    _RET="$url"
+
+    letter1="${name:0:1}"
```

Comment:
For `lib*` packages the subdirectory is `lib<initial>`, see http://archive.ubuntu.com/ubuntu/pool/main/.